src/screens/MainApp/DiagnoseRequest/DiagnoseSubmit/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { TouchableImage } from '~/helpers/TouchableImage'
+import styles from './styles'
+
+const DiagnoseSubmit = ({ onPress }) => (
+  <TouchableImage
+    style={styles.diagnoseSubmit}
+    onPress={onPress}
+    source={require('./resources/diagnoseSubmit.png')}
```

Comment:
Can you use `import` for this? I think you should be able to do it. What is the result of this `require`? In most bundlers, this returns the path where the image is located (i.e. a string). In that case you can `import imagePath from './path/to/image'`. The `import` / `require` is usually needed because bundlers change the location/names of the files.